aws/handlers/users_api_req_handler.go

```diff
@@ -0,0 +1,31 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/sal-org/sal-backend/aws/database"
+	"github.com/sal-org/sal-backend/aws/repositories"
+	"github.com/sal-org/sal-backend/aws/utils"
+	"github.com/sal-org/sal-backend/models"
+)
+
+// HandlePatientsRequest will handle all users gate api requests
+func HandleUsersRequest(req events.APIGatewayProxyRequest) (*events.APIGatewayProxyResponse, error) {
+	switch req.HTTPMethod {
+	case "GET":
+		db, err := database.CreateDB()
+
+		if err != nil {
+			return utils.GatewayResponse(http.StatusBadRequest, models.ErrorBody{ErrorMsg: aws.String(err.Error())})
+		}
+
+		repository := repositories.UsersRepository{DB: db}
```

Comment:
i think we should probably prevent fetching all users and allow getting user by id or certain/specific criteria or something.. its ok for now